.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+    "search.exclude": {
+        "**/vendor": true,
```

Comment:
you are right, I thought that this would override it.
I would like to keep `vendor` though, I often search for Node files in quick open to quickly see the properties of a particular Node class.